### DISCUSSION TITLE
Where are notes stored?

### DISCUSSION THREAD

**Discussion by avalonv** [koreader/koreader#9765] [Q&A] [answered]
Is there a way of seeing all my notes in one place?

**Comment by jokroese** [comment, marked as answer]
For any of the books you've made notes in, there will be a folder in the root directory called `[name of the book].sdr`. Inside that, there's a .lua file which has your notes.

There are some notes on how to export them all here: https://koreader.rocks/user_guide/#exportinghighlights

**Comment by jonnyl2** [comment]
On KOReader you can only directly view the notes of the currently opened book (under Bookmarks).
You can export multiple books' notes and then open the exported file in KOReader.

You can also use the software [KoHighlights](https://github.com/noembryo/KoHighlights) to view and export all your books' notes on your computer.
